### PR TITLE
Integration of Checkout view with Shipping and Succesful order pages

### PR DESCRIPTION
**Pull request by CxrlosKenobi**
## Description
On this PR, instead of split the `CartDrawer` component, I thought would be more convenient to build a whole new page for the checkout process, from getting the information about the customer until giving the thanks for a purchase.

**Note**
- The base code in the PR it's done, even though I had some troubles while trying to handle the multiples inputs of the user with React; I didn't include the input validation for the form inside the checkout, so that you guys can move forward in your respective branches. 

So basically what's left here is to get the input of the user in the forms and apply some fancy styles.

### CODE REVIEW

**Review by ChrisDeCleene — APPROVED** [review]
Great work on the Checkout Process, Carlos!

After walking through all of the updates, everything I saw made sense. When I ran it on my local server, I did notice that there were styling overlaps coming down from the orderCompleted component so I updated the selector to be a class of main instead of selecting all main jsx elements.

As far as validation, I figured out where the issue was passing around inputs. When using "setInputs", you just needed to pass in prevState instead of using spread operator directly on inputs (line 8 of Form.js). Also, since you render the Components on checkout via useEffect, it just needed to also have a dependency of inputs and then everything worked fine.

I added a function to the utilities file for form validation and plugged it into both the Checkout and Shipping Components. Besides that, I just added some additional props to each input so that it would update state properly and change with user changes.

Let me know if you have any questions about any of these changes or my comments.

Great work on this branch! It looks really good!
